src/fe_image.cpp

```diff
@@ -1000,6 +1005,14 @@ bool FeTextureContainer::is_swf() const
 	return m_swf;
 }
 
+float FeTextureContainer::get_movie_aspect_ratio() const
+{
+	if ( m_movie )
+		m_movie->get_aspect_ratio();
```

Comment:
I think you want a return here so it is "return m_movie->get_ar()"
Otherwise I think this PR looks good, thanks

Reply:
Suprisingly it worked. I think the compiler assumed I want a return and did it for me :) Thanks.